appveyor.yml

```diff
@@ -1,5 +1,5 @@
 init:
-  - git config --global core.autocrlf true
+  - git config --global core.autocrlf input
```

Comment:
This file is used only on Windows. Why is this change required? Will we need to change the configuration of other (non-AppVeyor) CI builds? (We don't use AppVeyor for official builds.)

If Windows users can't use `git config core.autocrlf true` after this change, expect resistance.